ui/src/lib/cookies.ts

```diff
@@ -0,0 +1,5 @@
+import Cookies from 'js-cookie';
+
+export default Cookies.withAttributes({
```

Comment:
If I read the docs on js-cookie correctly, just having this code ensures all cookies have same-site attribute? Even if it isn't necessarily used anywhere?

Reply:
That's my understanding, at least all cookies generated with this instance of "Cookies"